cmd/updater/main.go

```diff
@@ -150,9 +150,9 @@ func _main(ctx context.Context, timeNow func() time.Time) int {
 	runner := update.NewRunner(db, updater, ipGetter, p.cooldown, logger, timeNow)
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
-	forceUpdate := make(chan struct{})
-	go runner.Run(ctx, p.period, forceUpdate)
-	forceUpdate <- struct{}{}
+
+	go runner.Run(ctx, p.period)
+	runner.ForceUpdate()
```

Comment:
this hangs, should we replace this by something like `go runner.ForceUpdate(errors)` ? and check that error callback?

Reply:
That was because I forgot to create the `forceResult` channel, it's now fixed 😄 Thanks for finding this out!

And it's now running in a goroutine as we indeed don't want to block the rest of the goroutines with the first round of updates, especially since errors are logged within the updater so we don't need to collect errors.

Reply:
hahaha, I should have checked the constructor ^^. I'm glad I could help